components/table/ExpandableRow.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * ExpandableRow.jsx
+ * Created by Lizzie Salita 5/14/20
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const propTypes = {
+    data: PropTypes.object,
+    columns: PropTypes.array,
+    oddClass: PropTypes.string
+};
+
+const ExpandableRow = ({ data, columns, oddClass }) => {
+    const [expanded, setExpanded] = useState(false);
+    const icon = expanded ? 'chevron-down' : 'chevron-right';
+    const columnTitles = columns.map(({ title }) => title);
+    return (
+        <>
+            <tr className={`usda-table__row${oddClass} usda-table__row_expandable`}>
+                {columnTitles.map((col, i) => {
+                    if (col === 'name' && data.children) {
+                        return (
+                            <td
+                                key={`${data.name}`}
+                                className="usda-table__cell">
+                                <button
+                                    className="usda-table__expand-button"
+                                    onClick={() => setExpanded(!expanded)}>
```

Comment:
I think this creates a new function on render, maybe we could move this out of the return and pass the expanded property as the value of the button?